### DISCUSSION TITLE
Warning message to that local settings have been reset

### DISCUSSION THREAD

**Discussion by nvarghese-rm** [SafeExamBrowser/seb-mac#186] [Q&A]
We have installed version  SEB Windows 3.0 and find we keep getting this warning prompt every time we open the browser.

![image-20220224-101940](https://user-images.githubusercontent.com/87759879/155969128-8efd442a-cec2-473c-8920-8e50b9fbd4cf.png)

Once you click on OK, everything seems normal and we have no issues sitting the exam.

Any thoughts on what may be causing this issue?

**Comment by danschlet** [comment]
The screen shot is from the Mac version of SEB. But it is a customized SEB version, so you have to ask the provider of this custom SEB version for help. We strictly only support the original SEB version.

**Reply by nvarghese-rm** [reply]
Sorry, you are correct.  I wrongly entered it in Windows discussion and got the versions all  wrong too.

Yes, you are correct the image I added was from our custom version. 

But we see the same issue with the original version 3.0 now.
![MicrosoftTeams-image (131)](https://user-images.githubusercontent.com/87759879/155974094-c9272fd9-87a8-4257-bf94-dc86f85ab50c.png)

Any hints what and where the issue might be.
